tools/go-cli/go-whisk-cli/commands/action.go

```diff
@@ -351,11 +302,7 @@ var actionListCmd = &cobra.Command{
         return whiskErr
       }
       client.Namespace = qName.namespace
-    } else if len(args) > 1 {
-      whisk.Debug(whisk.DbgError, "Action list command must not have more than one argument\n")
-      errMsg := fmt.Sprintf("Invalid argument(s): %s.", strings.Join(args[1:], ", "))
-      whiskErr := whisk.MakeWskError(errors.New(errMsg), whisk.EXITCODE_ERR_GENERAL,
-        whisk.DISPLAY_MSG, whisk.DISPLAY_USAGE)
+    } else if whiskErr := checkArgs(args, 0, 1, "Action list", ""); whiskErr != nil {
```

Comment:
Need an "argument" error message (i.e. if too many args are specified)...   "An optional namespace argument is the only valid argument"